chain/client/src/client_actor.rs

```diff
@@ -543,6 +543,10 @@ impl Handler<Status> for ClientActor {
                     return Err(format!("No blocks for {:?}.", elapsed));
                 }
             }
+
+            if self.client.sync_status.is_syncing() {
+                return Err("Not healthy. Syncing!".to_string());
```

Comment:
Can it just be "The node is syncing", without "Not healthy" and the exclamation mark?

Reply:
done